Library/Formula/kobalt.rb

```diff
@@ -0,0 +1,46 @@
+class Kobalt < Formula
+  desc "Build system"
+  homepage "http://beust.com/kobalt"
+  url "https://github.com/cbeust/kobalt/releases/download/0.633/kobalt-0.633.zip"
```

Comment:
Yes. The exception we make is if this is Java code; it's OK for us to not build from source there because the "binaries" work cross-platform.